editor.planx.uk/src/pages/FlowEditor/components/Settings/GeneralSettings/ContactForm.tsx

```diff
@@ -0,0 +1,83 @@
+import { useFormik } from "formik";
+import React, { ChangeEvent } from "react";
+import InputDescription from "ui/editor/InputDescription";
+import Input from "ui/shared/Input";
+import InputRow from "ui/shared/InputRow";
+import InputRowLabel from "ui/shared/InputRowLabel";
+import * as Yup from "yup";
+
+import { SettingsForm } from "../shared/SettingsForm";
+import { FormProps } from ".";
+
+export default function ContactForm({ formikConfig, onSuccess }: FormProps) {
+  const formSchema = Yup.object().shape({
+    helpEmail: Yup.string()
+      .email("Please enter valid email")
+      .required("Help Email is required"),
+    helpPhone: Yup.string().required("Help Phone is required"),
+    helpOpeningHours: Yup.string(),
+  });
+
+  const formik = useFormik({
+    ...formikConfig,
+    validationSchema: formSchema,
+    onSubmit(values, { resetForm }) {
+      onSuccess();
+      resetForm({ values });
+    },
+  });
+
+  const onChangeFn = (type: string, event: ChangeEvent<HTMLInputElement>) =>
+    formik.setFieldValue(type, event.target.value);
+
+  return (
+    <SettingsForm
+      legend="Contact Information"
+      formik={formik}
+      description={
+        <InputDescription>
+          Details to help direct different messages, feedback, and enquiries
+          from users.
+        </InputDescription>
+      }
+      input={
+        <>
+          <InputRow>
+            <InputRowLabel>
+              Help Email
+              <Input
+                name="helpEmail"
+                onChange={(event) => {
+                  onChangeFn("helpEmail", event);
+                }}
+              />
+            </InputRowLabel>
+          </InputRow>
+          <InputRow>
+            <InputRowLabel>
+              Help Phone
+              <Input
+                name="helpPhone"
+                onChange={(event) => {
+                  onChangeFn("helpPhone", event);
+                }}
+              />
+            </InputRowLabel>
+          </InputRow>
+          <InputRow>
+            <InputRowLabel>
+              Help Opening Hours
```

Comment:
nit(ui/ux): I think we can remove the "Help..." prefixes from these labels - "Phone number", "Opening hours" and "Contact email address" feel like clearer / simpler labels. 

There's always room for the data model and it's representation to the user to be different 👍

Reply:
Cool!

What do you think about keeping the data model fields with the "help" prefix to group and indicate use?

Reply:
Yep, I'd keep that internally, and just change the labels to make most sense to users 👍